Source/control.cpp

```diff
@@ -114,29 +114,35 @@ bool IsRightPanelOpen()
 	return invflag || sbookflag;
 }
 
+constexpr int SdlRectYAdjustment = 16;

```

Comment:
I don't think this is related to SDL, in other places we are able to make SDL rectangles without any adjustment